src/plasma/plasma_manager.cc

```diff
@@ -940,6 +948,20 @@ int fetch_timeout_handler(event_loop *loop, timer_id id, void *context) {
        it != manager_state->fetch_requests.end(); it++) {
     FetchRequest *fetch_req = it->second;
     if (fetch_req->manager_vector.size() > 0) {
+      if(is_object_received(manager_state, fetch_req->object_id)){
+        int64_t duration = current_time_ms()
+                           - manager_state->received_objects[fetch_req->object_id];
+        if(duration > 10 * RayConfig::instance().manager_timeout_milliseconds()){
```

Comment:
I'd prefer not to hardcode this value `10` here, maybe add it to `ray_config.h`? What considerations should affect the value of this constant?

Reply:
The primary consideration is the time interval `delta` after the plasma manager receives an object and before the plasma manager receives the message which triggers  `process_add_object_notification` for the object received from the plasma store. `10 * RayConfig::instance().manager_timeout_milliseconds()` can be thought of as an upper bound on `delta`, after which the plasma manager assumes the receive has failed. I don't think this is necessary given @stephanie-wang's feedback.